data/dashboard.yml

```diff
@@ -68,7 +66,6 @@
         - govuk-rfcs
         - rubocop-govuk
         - publishing-e2e-tests
-        - styleguides
```

Comment:
Would it make sense to link to GDS Way in the intro (which is where this now lives)?

Reply:
Good shout. I've added on a commit.